docs/user/adding-new-metrics.md

```diff
@@ -50,7 +50,32 @@ The details of the metric parameters are described in [metric parameters](metric
 
 The `metrics.yaml` file is used to generate `Kotlin` code that becomes the public API to access your application's metrics.
 
-## Metric naming
+## Recommendations for defining new metrics
+
+“There are only two hard things in Computer Science: cache invalidation and naming things” -- attributed to Phil Karlton.
```

Comment:
Maybe this is too cute, but literally these are the two hard things in the recommendations.  Probably won't scale when we have more recommendations.

Reply:
... in which case we simply add the following addendum to the quote:

> and off-by-one errors.